docs/monthly-meeting/index.rst

```diff
@@ -11,4 +11,4 @@ Monthly reports in reverse chronological order.
    :maxdepth: 1
    :caption: Monthly reports
 
-   May 2021 (TBD) <2021-05.md>
\ No newline at end of file
+   Feb 2022 <2022-02.md>
```

Comment:
Can we instead add a `:glob:` option and add a single * here?

That, combined with a document heading change, would mean that future meeting notes PRs will only need to just add a file.

Reply:
Sounds good! I'll leave it to a future PR, maybe someone with more Sphinx knowledge will take it :)